compiler-rt/lib/scudo/standalone/tests/memtag_test.cpp

```diff
@@ -120,7 +120,15 @@ TEST_F(MemtagTest, SelectRandomTag) {
     uptr Tags = 0;
     for (uptr I = 0; I < 100000; ++I)
       Tags = Tags | (1u << extractTag(selectRandomTag(Ptr, 0)));
-    EXPECT_EQ(0xfffeull, Tags);
+    // std::popcnt is C++20
+    int PopCnt = 0;
+    while (Tags) {
+      PopCnt += Tags & 1;
+      Tags >>= 1;
+    }
+    // Random tags are not always very random, and this test is not about PRNG
+    // quality.  Anything above half would be satisfactory.
+    EXPECT_GE(PopCnt, 8);
```

Comment:
Can't you just replace this with __builtin_popcount(Tags)? I'm pretty sure llvm supports it.

Reply:
I'd rather not create a potential compiler dependency to save 2 lines of code in a test.

Reply:
Do you expect that this code will be compiled by something other than clang or gcc?

I don't feel very strongly about this, so leaving it like this is fine with me.